### DISCUSSION TITLE
Comment gerer le champ "SIRET de votre cuisine centrale" ?

### DISCUSSION THREAD

**Discussion by alemangui** [betagouv/ma-cantine#1101] [Question / réponse]
Voulons-nous que l'utilisateur puisse renseigner la cuisine centrale qui fournit ses repas ? Si oui, sous quelle forme ?

**Comment by valeriemerle** [comment]
Si je comprends bien la questions, je dirai que non. La cuisine centrale possède les données relatives à la fabrication de repas de ses satellites parfois mieux que les satellites eux mêmes.
